### PR TITLE
Mat 6728 single population excel

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6728](https://jira.cms.gov/browse/MAT-6728)
(Optional) Related Tickets:

### All Submissions

- [x] This PR has the JIRA linked.
- [x] Required tests are included.
- [x] No extemporaneous files are included (i.e Complied files or testing results).
- [x] This PR is merging into the **correct branch**.
- [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
- [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps

If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

- [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
- [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers

By Approving this PR you are attesting to the following:

- Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
- The tests appropriately test the new code, including edge cases.
- If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/services/export.service.ts [hunk 1]
```diff
@@ -10,17 +10,32 @@ import {
...
 
     //Generate other worksheets as needed
+    const groupNumber = testCaseExcelExportDtos[0].groupNumber;
+    const populationWorksheet = workbook.addWorksheet(
+      `${groupNumber} - Population Criteria Section`,
+    );
+    this.generatePopulationWorksheet(
+      populationWorksheet,
+      testCaseExcelExportDtos[0],
+    );
```
Comment (chubert-sb) [inline src/services/export.service.ts:38]: This is for single test case and single population for now and you are working on the multi right?

File: src/services/export.service.ts [hunk 2]
```diff
@@ -121,4 +136,165 @@ export class ExportService {
...
+  ) {
+    let foundPopulation: PopulationDto = null;
+    result.populations?.forEach((currentPopulation) => {
+      if (currentPopulation.name === population.name) {
+        foundPopulation = currentPopulation;
+      }
+    });
+    testCaseData.push(foundPopulation?.expected, foundPopulation?.actual);
+    return foundPopulation;
+  }
```
Comment (chubert-sb) [inline src/services/export.service.ts:209]: I might be getting confused on this, but do we need to loop here if we are already looping on the populations where this is being called?

File: src/services/export.service.ts (hunk 1 above)
Comment (sb-cecilialiu) [reply src/services/export.service.ts:38]: @chubert-sb correct.

File: src/services/export.service.ts (hunk 2 above)
Comment (sb-cecilialiu) [reply src/services/export.service.ts:209]: @chubert-sb Yeah this is needed, because before this method is called, there is a filtering getPopulations() method which gets the populations that have population values, so we need to populate the real values from the original dto.

**Review by chubert-sb — APPROVED** [review]
